styles.css

```diff
@@ -208,6 +214,11 @@ p, .email-address {
     html {
         font-size: 70%;
     }
+    .photo-rosa {
```

Comment:
In between the resolution break the image doesn't behave how I imagine you want it to. At the narrowest res it looks good but between full desktop and midway the image resizes small. Might want to investigate.

Reply:
So far as I can tell the image is still mis-behaving at the in between resolution. About half the width of my screen.